config.yaml

```diff
@@ -149,16 +149,23 @@ commands:
             sales_order_status_history
             sales_order_tax
             sales_order_tax_item
+          sales_invoice sales_invoice_* sales_invoiced_*
```

Comment:
Perhaps a bit of indentation so future edits are less likely to fragment, compare with sales_order*